src/main_backend.py

```
@@ -136,17 +175,23 @@ def shutdown(self, restart_service: bool = False) -> None:
             restart_service: If True, trigger a systemctl restart of the service after shutdown.
         """
         if restart_service:
-            print("Restarting systemctl service...")
+            print(f"{Colors.CYAN}Restarting systemctl service...{Colors.RESET}")
             try:
                 # Get the service name from environment or use a default
                 service_name = os.environ.get("SERVICE_NAME", "eagleeye")
                 result = os.system(f"sudo systemctl restart {service_name}")
```

Ruff (0.14.0)

Starting a process with a shell, possible injection detected

(S605)